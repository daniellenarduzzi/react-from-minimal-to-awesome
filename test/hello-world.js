import React from 'react'
import HelloWorld from '../src/hello-world'
import renderer from 'react-test-renderer'

test('HelloWorld match snapshot', () => {
    const component = renderer.create( 
        < HelloWorld/>
    )
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
})