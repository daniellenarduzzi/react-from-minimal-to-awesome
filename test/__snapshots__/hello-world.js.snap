// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld match snapshot 1`] = `
<h1>
   Hello World, React! 
</h1>
`;
